launch/model_bundle.py

```diff
@@ -12,12 +15,41 @@ class ModelBundle:
     """
 
     name: str
+    """
+    The name of the bundle. Must be unique across all bundles that the user owns.
+    """
+
     bundle_id: Optional[str] = None
```

Comment:
tbh I'm not sure why we're exposing `bundle_id` (since it's some identifier on our postgres db iirc)

Reply:
Yeah this is not ideal, though I wanted to keep this change to docs only. Didn't want to accidentally result in a breaking change 😬